### PR TITLE
Revert "Use 0.11.1 version of the flex-microform javascript so that incorrect…"

### PR DESCRIPTION
**Pull request by mulby**
Reverts edx/frontend-app-payment#442

### CODE REVIEW

**Review by inventhouse — APPROVED** [review]
What are the benefits vs. risks of unpinning this now?  I suggest we hold this until we have monitoring of the microform releases, but up to you, the change itself is fine.

**Comment by mulby** [comment]
Ah yes, should have linked to the alert I setup:

https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJzeW50aGV0aWNzLW5lcmRsZXRzLm1vbml0b3Itb3ZlcnZpZXciLCJpc092ZXJ2aWV3Ijp0cnVlLCJyZWZlcnJlcnMiOnsibGF1bmNoZXJJZCI6InN5bnRoZXRpY3MtbmVyZGxldHMuaG9tZSIsIm5lcmRsZXRJZCI6InN5bnRoZXRpY3MtbmVyZGxldHMubW9uaXRvci1saXN0In0sImVudGl0eUlkIjoiT0RneE56aDhVMWxPVkVoOFRVOU9TVlJQVW53NFl6aGhZV0V6WWkweE1EazNMVFEzTWpJdE9HWm1NeTA1WWpoa1lUaGlOamxrWldZIn0=&sidebars[0]=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5hY3Rpb25zIiwiZW50aXR5SWQiOiJPRGd4TnpoOFUxbE9WRWg4VFU5T1NWUlBVbnc0WXpoaFlXRXpZaTB4TURrM0xUUTNNakl0T0dabU15MDVZamhrWVRoaU5qbGtaV1kiLCJzZWxlY3RlZE5lcmRsZXQiOnsibmVyZGxldElkIjoic3ludGhldGljcy1uZXJkbGV0cy5tb25pdG9yLW92ZXJ2aWV3IiwiaXNPdmVydmlldyI6dHJ1ZX19&platform[accountId]=88178&platform[timeRange][duration]=1800000&platform[$isFallbackTimeRange]=true

Basically new relic will download the SDK every hour and check that the response contains the string "Flex Microform v0.11.1" if it doesn't it triggers the alert, I wanted to discuss with you and Emma where/how the alert should be routed. I tested it with my email, but presumably we want it to go to New Relic as a CAT-2 or something.

**Comment by inventhouse** [comment]
Neat!  If we can just put it on a dashboard somehow or something like that, mainly I want something we can easily see if the error rate goes nuts again
